pkg/ingester/ingester.go

```diff
@@ -1300,6 +1304,13 @@ func (i *Ingester) Push(ctx context.Context, req *cortexpb.WriteRequest) (*corte
 			})
 		}
 
+		if i.cfg.ActiveSeriesMetricsEnabled && succeededHistogramCount > oldSucceededHistogramsCount {
+			db.activeSeries.UpdateSeries(tsLabels, tsLabelsHash, startAppend, func(l labels.Labels) labels.Labels {
+				// we must already have copied the labels if succeededHistogramCount has been incremented.
+				return copiedLabels
+			})
+		}
```

Comment:
Can we combine this part with the code above so that we only update series once?

```
shouldUpdateSeries := (succeededSamplesCount > oldSucceededSamplesCount) || (succeededHistogramCount > oldSucceededHistogramsCount)
if i.cfg.ActiveSeriesMetricsEnabled && shouldUpdateSeries {...}
```

Reply:
Can we introduce a variable instead? To avoid the line to be too long

Reply:
Thanks